File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;



namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }



        private void Form1_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'carsDBDataSet.Customers' table. You can move, or remove it, as needed.
            this.customersTableAdapter.Fill(this.carsDBDataSet.Customers);
            this.ordersTableAdapter1.Fill(this.carsDBDataSet.Orders);
            myGridControl1.DataSource = carsDBDataSet.Tables["Customers"];
        }




        private void myGridControl1_SaveButtonClick(object sender, DevExpress.MyControl.MyGridView e)
        {
            MessageBox.Show(string.Format("SaveButtonClick at MyGridView with {0} rows", e.RowCount));
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Windows.Forms



Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub



		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'carsDBDataSet.Customers' table. You can move, or remove it, as needed.
			Me.customersTableAdapter.Fill(Me.carsDBDataSet.Customers)
			Me.ordersTableAdapter1.Fill(Me.carsDBDataSet.Orders)
			myGridControl1.DataSource = carsDBDataSet.Tables("Customers")
		End Sub




		Private Sub myGridControl1_SaveButtonClick(ByVal sender As Object, ByVal e As DevExpress.MyControl.MyGridView) Handles myGridControl1.SaveButtonClick
			MessageBox.Show(String.Format("SaveButtonClick at MyGridView whith {0} rows", e.RowCount))
		End Sub
	End Class
End Namespace